framework/src/modules/chain/blocks/block_version.js

```diff
@@ -19,7 +19,20 @@
  *
  * @property {number} currentBlockVersion - Current block version used for forging and verify
  */
-const currentBlockVersion = 1;
+const currentBlockVersion = 2;
+
+const exceptions = {
+	blockVersions: {
+		'1': {
+			start: 1,
+			end: 7,
+		},
+		'2': {
+			start: 8,
+			end: 6901027,
+		},
+	},
+}; // TODO BFT: define strategy to inject this variable from chain config
```

Comment:
These exceptions comes from the configuration, and these will be different for each network, so you can't use static definition here.